### ISSUE TITLE
Add orthologous mapping (between species).

### ISSUE THREAD

**Issue by DeniseSl22** [bridgedb/Goliath#16]
orthologous mapping (between species) for genes/proteins would be great for people that want to do transcriptomics pathway analysis on a different species. For example they have Mouse IDs, and want to check how their result look on Human IDs. I think BioMart has tools for this, so this would really be an enhancement (and it is not relevant for chemicals).

**Comment by Chris-Evelo** [comment]
The question is do we want to support it twice? We also provide homology-based pathway species conversion (which should be added as an on-request thing to the species pathway overview. We currently do this based on homologene and I just asked Kristina and Martina to see whether we should use PANTHER instead, since that is what Reactome is using. As far as your example goes, mouse and human pathways once had this same conversion done and since then updates should have been added to either both species (which I agree typically did not happen), or be species specific.

**Comment by DeniseSl22** [comment]
Yes, but you could also think of other species then Mouse; I think @mkutmon does conversions for mouse and rat to human on a semi regular basis, but that doesn't mean it works the other way around. I think that regardless of vertebrates species, one should be able to perform pathway analysis. And we could point people to other tools when we don't have a specific feature; but then you will loose them as users. I think this is not a feature we would need right away, but would be nice to have at some point.
